src/theme.jl

```diff
@@ -58,88 +58,127 @@ function get_stroke_vector(linestyle::Symbol)
   error("unsupported linestyle: ", linestyle)
 end
 
+using DocStringExtensions
+
+"""
+$(FIELDS)
+"""
```

Comment:
:tada: 

This will make keeping the docs in sync much easier and avoid issues like https://github.com/GiovineItalia/Gadfly.jl/issues/852